### ISSUE TITLE
Translation when user denied permissions

### ISSUE THREAD

**Issue by veselinovicst** [jitsi/lib-jitsi-meet#1541]
Hi there,

We are using an implementation of Jitsi in Dutch and have modified many translations in our customizations. However, there is one message in English that does not seem to be modifiable.

When a user rejects the permissions to use audio/video the following error message is shown.
This message is part Dutch and part English. We are able to configure the first part (Audio- en videofout:), but it does not seem to be possible to modify the second part (user denied permission to use device(es): audio, video.

Wondering if this is possible to change to Dutch as well?
The screenshot is from meet.jit.see, using a Chrome set to dutch

![sc1](https://user-images.githubusercontent.com/24300198/112999101-f05d2b80-916e-11eb-99a5-9d4fbec06bd9.png)

**Comment by damencho** [comment]
Please, when you have questions or problems use the community forum before opening new issues, thank you.

The screenshot is from jitsi-meet and your question is about jitsi-meet. 
All translations are here: https://github.com/jitsi/jitsi-meet/tree/master/lang and you can create PRs there with updated translations. It can be that some keys from main.json are missing from your language main file and need to be added.
We will have a translation service at some point, which will make translation easier and quicker and the missing keys will be handled automatically, but till then, this is a manual process.

**Comment by veselinovicst** [comment]
Thank you for answering.

I did ask on forum and answer was to open an issue here.

https://community.jitsi.org/t/translation-when-user-denied-permissions/94188

**Comment by damencho** [comment]
Yeah, you should mention that this was discussed with the link, thanks. 
And from the discussion, I got more details and see why this is open here, sorry.

**Comment by damencho** [comment]
There is no way to add a translation to this project. This needs to be handled in jitsi-meet as this strings also has logic checking browser state ...
